### DISCUSSION TITLE
Parser plugins

### DISCUSSION THREAD

**Discussion by cplerch** [foambubble/foam#1104] [Q&A]
Hi, I'm looking for a way to add extraPlugins (e.g. remark-math) to the remark markdown parser, without having to modify the code. Is there an easy way to do it with configuration settings?

**Comment by riccardoferretti** [comment]
I can't think of an easy way to achieve that atm, especially for what regards adding plugins (which would need to be part of the Foam build) vs disabling them. I am open to hearing ideas if there is any suggestion around this
